acceptance/tests/allow_arbitrary_node_name_for_apply.rb

```diff
@@ -1,5 +1,9 @@
 test_name "node_name_value should be used as the node name for puppet apply"
 
+tag 'audit:medium',
+    'audit:integration',  # Ruby level integration tests already exist. This acceptance test can be deleted.
+    'audit:delete'
```

Comment:
i'm fine either way. i'll only add that i think people want to see and be able to show results.  soon.